### PR TITLE
3914: docketRecord.action field ui

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: web-client/src/views/EditDocketRecordEntry/EditDocketEntryMetaModal.jsx [hunk 1]
```diff
@@ -95,21 +95,17 @@ export const EditDocketEntryMetaModal = connect(
...
-            htmlFor="servedAt"
-            id="document-served-at-label"
-          >
-            Served
+        <FormGroup errorText={validationErrors && validationErrors.action}>
+          <label className="usa-label" htmlFor="filedBy" id="action-label">
+            Action
           </label>
           <input
-            area-describedby="document-served-at-label"
```
Comment (matthopson) [inline web-client/src/views/EditDocketRecordEntry/EditDocketEntryMetaModal.jsx:107]: This is fine... :P

File: web-client/src/views/EditDocketRecordEntry/EditDocketEntryMetaModal.jsx [hunk 2]
```diff
@@ -119,23 +115,21 @@ export const EditDocketEntryMetaModal = connect(
...
-            id="document-served-parties-label"
+            htmlFor="servedAt"
+            id="document-served-at-label"
           >
-            Parties
+            Served
           </label>
           <input
-            area-describedby="document-served-parties-label"
+            area-describedby="document-served-at-label"
```
Comment (matthopson) [inline web-client/src/views/EditDocketRecordEntry/EditDocketEntryMetaModal.jsx]: Probably want to double-check all of these in this component. "area" => "aria"

**Review by matthopson — APPROVED** [review]
Need to fix the aria labels, otherwise, tasty-town.

File: web-client/src/views/EditDocketRecordEntry/EditDocketEntryMetaModal.jsx (hunk 2 above)
Comment (sutt0n) [reply web-client/src/views/EditDocketRecordEntry/EditDocketEntryMetaModal.jsx]: @matthopson I just pushed a commit for this. 😄

**Pull request merged**